server/tests/middlewares/createArticleValidation.spec.js

```diff
@@ -0,0 +1,68 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '<server>/app';
+import {
+  goodArticleData,
+  getArticleData,
+  emptyArticleData,
+  invalidTitle,
+  invalidCategoryId
+} from '<fixtures>/articleValidationData';
+
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+describe('Article creation validation', () => {
+  it('should return 400 on empty request body or if any of the field is empty', (done) => {
+    chai
+      .request(app)
+      .post('/api/v1/articles')
+      .send(emptyArticleData)
+      .end((err, res) => {
+        expect(res.status && res.body.status).to.be.equal(400);
+        expect(res).to.be.an('object');
+        expect(res.body)
+          .to.have.property('error');
+        expect(res.body.error.title).to.be.an('array').that.does.include('The title field is required.');
+        expect(res.body.error.body).to.be.an('array').that.does.include('The body field is required.');
+        expect(res.body.error.categoryId).to.be.an('array').that.does.include('The categoryId field is required.');
+        done();
+      });
+  });
+
+  it('should return true if categoryId is a valid UUID', () => {
+    chai
+      .expect(goodArticleData.categoryId).to.be.a.uuid();
```

Comment:
This test if the goodArticleData categoryId is a valid UUID

Reply:
I agree however I don't see chai-uuid being imported in this file. Can you please remove it and see if it still works.